app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationController.scala

```diff
@@ -454,4 +460,20 @@ class BusinessIdentificationController @Inject()(
       case Failure(responses) if responses.contains(InvalidEmail) => true
       case _                                                      => false
     }
+
+  def alreadyStarted: Action[AnyContent] = Action.async { implicit request =>
```

Comment:
normally we name GET pages with show... so this would be showAlreadyStarted but not a big issue